docs/Clearwater_stress_testing.md

```diff
@@ -12,82 +12,78 @@ package installed.
+Clearwater SIP stress nodes provide a set of SIPp scripts to run against your Sprout cluster. From Sprout's point of view, they look like a P-CSCF.  The nodes log their success/failure to `/var/log/clearwater-sip-stress` and also print a human-readable summary of the stress (with informatin about percentage of failed calls, average latency, and so on).
 
-The SIPp stress starts as soon as the node comes up - this is by design, to avoid you having to manually start 50+ nodes generating traffic.  If you want to stop them from generating traffic, turn the whole node down.
+They follow a fixed load profile, assuming 1.3 calls/hour for each subscriber (split equally between incoming and outgoing calls) and 2 re-registrations per hour.
 
-Each SIP stress node picks a single bono to generate traffic against.  This bono is chosen by matching the bono node's index against the SIP stress node's index.  As a result, it's important to always give SIP stress nodes an index.
+## Deploying a stress node
 
-### How they work
+Follow [this process](https://github.com/Metaswitch/crest/blob/dev/docs/Bulk-Provisioning%20Numbers.md) to bulk provision the number of subscribers you want. As a general guideline, we'd expect a small deployment (with one Sprout, Homestead and optionally a Ralf node, each with one full CPU core) to handle at least 50,000 subscribers.
 
-The clearwater-sip-stress package includes two important scripts.
+You should also ensure that the `reg_max_expires` setting is set to 1800 rather than the default of 300 - see our [configuration guide](Clearwater_Configuration_Options_Reference.md) for instructions. This matches the assumptions in our load profile of 2 re-registers per hour.
 
-* `/usr/share/clearwater/infrastructure/scripts/sip-stress`, which generates a `/usr/share/clearwater/sip-stress/users.csv.1` file containing the list of all subscribers we should be targeting - these are calculated from properties in `/etc/clearwater/shared_config`.
-* `/etc/init.d/clearwater-sip-stress`, which runs `/usr/share/clearwater/bin/sip-stress`, which in turn runs SIPp specifying `/usr/share/clearwater/sip-stress/sip-stress.xml` as its test script. This test script simulates a pair of subscribers registering every 5 minutes and then making a call every 30 minutes.
+If you're using chef, create your stress test node by typing `knife box create -E ENVIRONMENT sipp --index 1`.
 
-The stress test logs to `/var/log/clearwater-sip-stress/sipp.<index>.out`.
+Otherwise, follow these steps to deploy a stress node manually:
 
-## Running Stress
+* create a new virtual machine and bootstrap it [by configuring access to the Project Clearwater Debian repository](Manual_Install.md#configure-the-apt-software-sources).
+* set the following properties in /etc/clearwater/local_config:
+    * (required) local_ip - the local IP address of this node
+    * (optional) node_idx - the node index (defaults to 1)
```

Comment:
Why is this needed? (It feels like it should either not be needed at all, or it's required)